app/assets/javascripts/discourse-common/addon/utils/dom-utils.js

```diff
@@ -0,0 +1,51 @@
+function isWindow(obj) {
+  return obj != null && obj === window;
+}
```

Comment:
This can be reduced to `obj === window`. And that short snippet is better inlined, no need for a function. 🙂

Reply:
I did this after looking at jquery source code implementation, I do agree with you